python/jupytergis_qgis/jupytergis_qgis/qgis_loader.py

```diff
@@ -174,15 +214,38 @@ def qgis_layer_to_jgis(
             maxZoom=max_zoom,
             minZoom=min_zoom,
         )
+
+        renderer = layer.renderer()
+        styles = renderer.styles()
+        color = {}
+
+        for style in styles:
+            symbol = style.symbol()
+            geometry_type = style.geometryType()
+
+            opacity = symbol.opacity()
+            alpha = hex(int(opacity * 255))[2:].zfill(2)
+
+            # 0 = points, 1 = lines, 2 = polygons
+            if geometry_type == 0:
+                color["circle-fill-color"] = symbol.color().name() + alpha
+                color["circle-stroke-color"] = symbol.color().name() + alpha
+
+            if geometry_type == 1:
+                color["stroke-color"] = symbol.color().name() + alpha
+
+            if geometry_type == 2:
+                color["fill-color"] = symbol.color().name() + alpha
+
         # TODO Load source-layer properly, from qgis symbology?
         try:
             source_layer = get_source_layer_names(url)[0]
             layer_parameters["sourceLayer"] = source_layer
         except ValueError:
             pass
-        # TODO Load style properly
+
         layer_parameters.update(type="fill")
```

Comment:
Should we keep this?

Reply:
I'm pretty sure the `source_layer` stuff isn't being used anywhere right now, so it should be fine to get rid of. The `type` is required in the schema so we should leave that for now.

Reply:
Ok! Thanks for your answer

Let's merge this then!